src/lib.rs

```diff
@@ -135,17 +135,19 @@ where
     // unsafe { asm!("" : : "r"(&PERSIST_MARKER)) }; // hack used in nightly's back_box(), requires feature asm
     // unsafe { asm!("" : : "r"(&DEFERED_MARKER)) };
 
-    // sets panic hook to abort
-    std::panic::set_hook(Box::new(|_| {
-        std::process::abort();
-    }));
+    if hook {
+      // sets panic hook to abort
+      std::panic::set_hook(Box::new(|_| {
+          std::process::abort();
+      }));
+    }
 
     let mut input = vec![];
 
     // initialize forkserver there
     unsafe { __afl_manual_init() };
 
-    while unsafe { __afl_persistent_loop(1000) } != 0 {
+    while unsafe { __afl_persistent_loop(1) } != 0 {
```

Comment:
@pedrocr Do you remember why this change was made?

Reply:
Unfortunately I forget why that was needed. Not sure if it just didn't work without that or if it was just because of performance. I can't find any docs for that function either to even know what the parameter does.

Reply:
I've just tested it and it works without this. I've pushed a version that rebased to master and doesn't have this change. The diff looks pretty clean now.

Reply:
I don't know for sure, but I suspect that the parameter is how frequently the process is restarted during in-process fuzzing, in number of executions before restart. So if you set the parameter to 1 it should be noticeably slower, almost equivalent to the performance of stdin mode.